ERROR in Cannot find module 'prettier'

Hi! I just downloaded a version of the wd_s starter theme and when I try to run a build I get this error: 
_ERROR in Cannot find module 'prettier'._

Not sure if this is a problem on my end. 

I tried installing prettier locally using: npm install --save-dev --save-exact prettier

But then I start getting this error: 
_ERROR in Cannot read property 'sync' of undefined_

Does prettier need to be installed globally?

Update - I thought the version of npm I had installed  was up to date, but it was not. Updated that and restarted VS Code. 

Now getting an new error when I try and run npm i --legacy-peer-deps.

_@arkweid/lefthook
npm ERR! command failed_

--- a/package.json
+++ b/package.json
@@ -40,7 +40,7 @@
 		"npm-run-all": "^4.1.5",
 		"postcss-loader": "4.2.0",
 		"postcss-preset-env": "^6.7.0",
-		"prettier": "npm:@wordpress/prettier-config",
+		"prettier": "npm:wp-prettier@^2.2.1-beta-1",
 		"sass-loader": "10.2.0",
 		"stylelint-config-prettier": "^8.0.2",
 		"stylelint-webpack-plugin": "^2.2.2",

